docs/development/setup_development.rst

```diff
@@ -122,7 +122,7 @@ different version, the path to ``virtualenvwrapper.sh`` will differ. Running
 .. code:: sh
 
     sudo easy_install pip # if you don't already have pip
-    pip install -U virtualenvwrapper
+    sudo -H pip install -U virtualenvwrapper --ignore-installed six
```

Comment:
Why do you need `sudo`?

Reply:
This will not work without `sudo`. You get a permission denied error.  That's why you pass the `-H` flag, to install in the users' home directory. Correct me if I'm wrong @redshiftzero but I think pip recommends _not_ using sudo with pip but in this case I think the correct behavior is to use `sudo` and `-H`.

I think this is the upstream [issue](https://github.com/pypa/pip/issues/3165)

Reply:
The traceback in #2698 showed attempts to write the pip libs to `/Library/Python/2.7/site-packages/`, thus the root perms. Personally, I use `pip install --user` to avoid having to elevate perms for pip libs. Not sure that'll work out of the box on macOS, though.

Might be worthwhile to recommend a homebrew workflow, if possible.

The `--ignore-installed six` strikes me as a more glaring problem, particularly because there's no explanation for it.

Reply:
@conorsch I looked for a homebrew solution for pip install virtualenvwrapper and couldn't easily find one.

Reply:
Did you try `brew install pyenv-virtualenvwrapper`? I think that should work and obviate the need for `sudo pip`

Reply:
Must defer to folks using macOS full-time, but I vastly prefer a homebrew workflow for bootstrapping the dev env. Reasons homebrew is rad:

* common for developers to have it already
* setup docs can be linked to, so we don't have to maintain
* no sudo required to install pip libs (warrants further testing)

Let's shoot for a homebrew solution and fall back from there if possible. @redshiftzero I think you're honestly the best candidate for nailing these steps down, mind creating a fresh env—possibly in a new user—and running through the doc?

Reply:
Fair, I'll put this in my queue for today